### DISCUSSION TITLE
Implementing holding gesture for 2x speed

### DISCUSSION THREAD

**Discussion by jboix** [SRGSSR/pillarbox-apple#747] [Q&A] [answered]
I am currently working on implementing a long press gesture, similar to the functionality in the YouTube app that allows users to increase the playback speed to 2x by holding down.

I've looked through the documentation and existing codebase, but I'm not sure which is the best approach to implement this feature. If anyone has experience or insights into handling gestures, I would greatly appreciate your input.

**Comment by defagos** [comment, marked as answer]
This has already been implemented in the demo. Please see #746.
